src/libcommon/info/errorinfo.cpp

```diff
@@ -21,10 +21,9 @@
 namespace KDC {
 
 ErrorInfo::ErrorInfo() :
-    _dbId(0), _time(0), _level(ErrorLevel::Unknown), _functionName(QString()), _syncDbId(0), _workerName(QString()),
-    _exitCode(ExitCode::Unknown), _exitCause(ExitCause::Unknown), _localNodeId(QString()), _remoteNodeId(QString()),
-    _nodeType(NodeType::Unknown), _path(QString()), _conflictType(ConflictType::None),
-    _inconsistencyType(InconsistencyType::None), _cancelType(CancelType::None) {}
+    _time(0), _level(ErrorLevel::Unknown), _syncDbId(0), _exitCode(ExitCode::Unknown), _exitCause(ExitCause::Unknown),
+    _nodeType(NodeType::Unknown), _conflictType(ConflictType::None), _inconsistencyType(InconsistencyType::None),
+    _cancelType(CancelType::None) {}
```

Comment:
Why not giving default values to all those variables in the header file?

Reply:
Yes, done.